src/main/java/uk/gov/hmcts/ecm/common/model/servicebus/tasks/UpdateDataTask.java

```diff
@@ -204,51 +205,86 @@ private void updateJudgement(CaseData caseData, JudgementType judgementType) {
         CaseJudgementUpdate.updateCaseWithJudgement(caseData, judgementType);
     }
 
-    private void updateRespondentRep(CaseData caseData, RepresentedTypeR representedType) {
-
-        //case 1: update is adding
-
-        //case 2: update is removing
+    private void updateRespondentRep(CaseData caseData, UpdateDataModel updateDataModel) {
+        RepresentedTypeR representedType = updateDataModel.getRepresentedType();
+        String isRespondentRepRemovalUpdate = updateDataModel.getIsRespondentRepRemovalUpdate();
```

Comment:
Move this within the following if statement as you aren't using it outside the if statement